webservice/pom.xml

```diff
@@ -173,6 +173,16 @@
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Currently the migrations are invoked from the cli arguments passed to StreamlineApplication class, that is why dependency is added to webservice module.